### PR TITLE
Included test for Edit a VM and add install guest agent option

### PR DESCRIPTION
**Pull request by yarunachalam**

### CODE REVIEW

File: harvester_e2e_tests/scenarios/test_create_vm.py [hunk 1]
```diff
@@ -206,6 +208,68 @@ def test_create_vm_do_not_start(request, admin_session, image, keypair,
...
+            break
+    upd_user_data = yaml.safe_load(cloudinit['cloudInitNoCloud']['userData']
+                                   .replace('\\n', '\n'))
+    assert 'packages' in upd_user_data
+    assert 'runcmd' in upd_user_data
+    assert 'qemu-guest-agent' in upd_user_data['packages']
+    assert 'systemctl' in upd_user_data['runcmd'][0]
+    assert 'enable' in upd_user_data['runcmd'][0]
+    assert '--now' in upd_user_data['runcmd'][0]
+    assert 'qemu-guest-agent' in upd_user_data['runcmd'][0]
```
Comment (guangyee) [inline harvester_e2e_tests/scenarios/test_create_vm.py:270]: On openSUSE, the name of the guest agent service is `qemu_ga` I think.

Comment (yarunachalam) [reply harvester_e2e_tests/scenarios/test_create_vm.py]: Done

File: harvester_e2e_tests/fixtures/vm.py [hunk 2]
```diff
@@ -60,12 +60,12 @@ def user_data_with_guest_agent(keypair):
       - %s
 package_update: true
 packages:
-  - qemu-guest-agent
+  - qemu-ga
```
Comment (guangyee) [inline harvester_e2e_tests/fixtures/vm.py:63]: The package name is correct, it should be `qemu-guest-agent`. Only the service name need to be `qemu-ga`.

**Review by guangyee — APPROVED** [review]
LGTM. We'll figure out the second part, which is to start with an image without qemu agent pre-installed in the next PR.

File: harvester_e2e_tests/fixtures/vm.py (hunk 2 above)
Comment (yarunachalam) [reply harvester_e2e_tests/fixtures/vm.py]: done

**Review by guangyee — APPROVED** [review]
lgtm

**Pull request merged**